### DISCUSSION TITLE
Ability to pin map to screen when walking (if this is possible)

### DISCUSSION THREAD

**Discussion by ArmoredArtist** [GoMudEngine/GoMud#94] [Ideas]
I would like to have the option to keep looking at the map when I walk without the game text pushing it upwards. Is it possible to somehow pin it and have it update as you move?

**Comment by Volte6** [comment]
I had to abandon some features that mudlet made impossible (like redrawing stuff). I had a cool status bar that would update for stuff like a progress bar or a % complete dynamically, but mudlet breaks it... likewise, any kind of drawing of the map to keep it on screen wouldn't be possible... would have to lean on the automapping ability of these programs, really.

**Reply by ArmoredArtist** [reply]
is it possible to add to the web client only with an auto detect of how your connecting to turn it on and off?

**Reply by Volte6** [reply]
Yeah I think I can add some sort of special thing like that eventually, dont' see why not

**Comment by Volte6** [comment]
I added an experimental webclient modal that shows up and updates when you look at rooms. It's just a simple thing to test the idea, and maybe to expand for future similar ideas.
